docs/mutant-minitest.md

```diff
@@ -64,7 +64,7 @@ This prints a report like:
 
 ```sh
 Mutant environment:
-Matcher:         #<Mutant::Matcher::Config match_expressions: [AUOM*]>
+Matcher:         #<Mutant::Matcher::Config subjects: [AUOM*]>
```

Comment:
I think this is much nicer to read. Its clear the domain of this method is matching, else the class would not be named `Mutant::Matcher`.

So the `match_` prefix was redundant always.

Just naming the key `expressions` would IMO also suck as it than would have a sibling `ignore_expressions` which does not clearly indicate what the difference between both would be.

Hence replacing this with `subjects` makes it very clear. And `subject_expressions` is kind of redundant as we can only match expressions.